src/reference/00-Getting-Started/11-Custom-Settings.md

```diff
@@ -239,7 +239,7 @@ lazy val library = (project in file("library")).
       val old = sampleStringTask.value
       println("stopping...")
       Thread.sleep(500)
-      old      
```

Comment:
There are two `sampleStringTask`s defined in this example, with the one referring to the other. Is that correct? Even if it is, would the example be clearer if we renamed one of those?